rotkehlchen/assets/asset.py

```diff
@@ -402,6 +402,61 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xBB0E17EF65F82Ab018d8EDd776e8DD940327B28b'): 'AXS',
 }
 
+WORLD_TO_UPHOLD = {
+    'BTC': 'BTC',
+    strethaddress_to_identifier('0xc4f33F15ed2F2C5F8D5B542dD30518A50D9f843f'): 'BTC0',
```

Comment:
Note: This asset is not supported at the moment. It's not in our global.db. A user can potentially add it on their own or we can add it in a next release.